app/lib/actions/pulp3/content_view_version/export.rb

```diff
@@ -0,0 +1,32 @@
+module Actions
+  module Pulp3
+    module ContentViewVersion
+      class Export < Pulp3::AbstractAsyncTask
+        input_format do
+          param :smart_proxy_id, Integer
+          param :content_view_version_id, Integer
+          param :export_base_dir, String
+        end
+
+        def export(cvv)
+          smart_proxy = SmartProxy.find(input[:smart_proxy_id])
```

Comment:
you can just call 'smart_proxy' here, Pulp3::AbstractTask defines the 'smart_proxy' method

Reply:
updated.